ruby-challenges.rb

```diff
@@ -12,6 +11,21 @@
 
 # Pseudo code:
 
+# create a function named even_odd with a parameter of a number
+def even_odd(number)
+  # if the remainder of a number divided by 2 is strictly equal to 0   
+  if number % 2 == 0
+    # return number is even
+    "#{number} is even"
+  else
+    # else return number is odd
+    "#{number} is odd"
+  end
+end 
+puts even_odd(reposts1)  # 7 is odd
+puts even_odd(reposts2)  # 42 is even
+puts even_odd(reposts3)  # 221 is odd
```

Comment:
don't forget about some of Ruby's built in methods that can make this even cleaner... hint hint